style.css

```diff
@@ -0,0 +1,79 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 10px;
+}
+
+body {
+  font-family: "Roboto", "Arial", sans-serif;
+  color: black;
+}
```

Comment:
##Font 
-[optional]Kindly as the figma design use the 'Inter' font for the text